app/hooks/wallet/MnemonicStorage.ts

```diff
@@ -13,20 +13,6 @@ export class MnemonicStorage {
   constructor (private readonly network: Network) {
   }
 
-  async getHdNodeProvider (): Promise<MnemonicHdNodeProvider> {
```

Comment:
You have re-arrange code enabled in your IDE? Should not rearrange codes

Reply:
my bad, it's the default setting of the plugin